git-credential-oauth.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: git-credential-oauth
+  version: 0.13.4
+  epoch: 0
+  description: "A Git credential helper that securely authenticates to GitHub, GitLab and BitBucket using OAuth"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - git
+
+environment:
+  environment:
+    CGO_ENABLED: 0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hickford/git-credential-oauth
+      tag: v${{package.version}}
+      expected-commit: 310d7b668ae9aca58999c95b14f1d95a6db6b045
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -s -w
+        -X main.version=v${{package.version}}
+      output: git-credential-oauth
+      packages: .
+
+  - uses: strip
```

Comment:
can be avoided because we already strip via go/build pipeline.

Reply:
Thanks for the tip and link to the `go/build` pipeline. I have removed this duplicate step from the pipeline